qlib/config.rs

```diff
@@ -54,6 +57,19 @@ impl Config {
     pub fn Async(&self) -> bool {
         return self.LogType == LogType::Async;
     }
+
+    //
+    // The assigned value is taken in consideration
+    // if it is at least equal to the required
+    // MIN_VCPU_COUNT value.
+    //
+    pub fn get_config_vcpu_amount(&self) -> Option<usize> {
+       if self.LimitVMvCPUAmount < MIN_VCPU_COUNT as u64 {
```

Comment:
Why is there such a limitation that the minimal vcpu count should be greater than 2 ?

Reply:
It sets the limit to at least two because vCPU_0 is used for [IOWait();](https://github.com/QuarkContainer/Quark/blob/483b83c1f51958f0b1cbc67667a2a311bfe39647/qkernel/src/lib.rs#L538)